real_time/surface_src/java_src/BuggyRos/src/test/java/NumberSink.java

```diff
@@ -0,0 +1,66 @@
+import java.util.concurrent.Semaphore;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import com.roboclub.robobuggy.ros.Message;
+import com.roboclub.robobuggy.ros.MessageListener;
+import com.roboclub.robobuggy.ros.Node;
+import com.roboclub.robobuggy.ros.Subscriber;
+
+public class NumberSink implements Node {
+
+	Subscriber s;
+	Semaphore sem = new Semaphore(1);
+	AtomicInteger count = new AtomicInteger(1);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2319545998/issues/source?bid=2938665&fileBranchId=3199623#l13)